### PR TITLE
[MU3] Export hidden tempo text to MusicXML

### PR DESCRIPTION
**Pull request by matangover**
Hidden tempo text is now exported as a `<sound tempo="XX"/>` element without a container `<direction>` element.

This is now in line with the way MuseScore imports `<sound>` elements, see: https://github.com/musescore/MuseScore/pull/7171

I changed testSound1 from a test with a reference to a roundtrip import-export test, because it is now fully supported. Some other test reference files had to be changed because they contain hidden tempo markings as well.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [x] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
rebase needed.

**Comment by matangover** [comment]
rebase done

File: mtest/musicxml/io/tst_mxml_io.cpp [hunk 1]
```diff
@@ -171,7 +171,7 @@ private slots:
       void slurTieLineStyle() { mxmlIoTest("testSlurTieLineStyle"); }
       void slurs() { mxmlIoTest("testSlurs"); }
       void slurs2() { mxmlIoTest("testSlurs2"); }
-      void sound1() { mxmlIoTestRef("testSound1"); }
+      void sound1() { mxmlIoTest("testSound1"); }
```
Comment (Jojo-Schmitz) [inline mtest/musicxml/io/tst_mxml_io.cpp:174]: Why no longer test against the ref file, but at the ssame tuime updating that in the same PR?

**Comment by Jojo-Schmitz** [comment]
I wonder whether this would need to get ported to master

**Pull request merged**